src/ssh.rs

```diff
@@ -192,16 +200,61 @@ fn create_ssh_config(
 ) -> Result<()> {
     let config_file = fs::OpenOptions::new()
         .write(true)
-        .create(true)
-        .truncate(true)
+        .create_new(true)
         .open(config_path.to_str().unwrap())
-        .map_err(|err| anyhow::anyhow!("Failed to open ssh config file: {err}"))?;
+        .map_err(|err| match err.kind() {
+            std::io::ErrorKind::AlreadyExists => {
+                anyhow::anyhow!(
+                    r#"ssh config file already exists and would be overwritten.
+Please remove config file first."#
+                )
+            }
+            _ => anyhow::anyhow!("Failed to create ssh config file: {err}"),
+        })?;
 
     let mut writer = BufWriter::new(config_file);
 
-    writeln!(
-        &mut writer,
-        r#"Host bastion
+    if let Ok("windows") = std::env::var("CONTAINER_HOST").as_deref() {
+        writeln!(
+            &mut writer,
+            r#"Host bastion
+	User {}
+	Hostname {}
+	Port {}
+	IdentityFile ~/.ssh/{}
+	CertificateFile ~/.ssh/{}
+	ProxyCommand none
+
+Host {}
+	User {}
+	IdentityFile ~/.ssh/{}
+	CertificateFile ~/.ssh/{}
+	ProxyCommand ssh bastion"#,
+            bastion_details.username,
+            bastion_details.hostname,
+            bastion_details.port,
+            bastion_details
+                .priv_key
+                .file_name()
+                .unwrap()
+                .to_str()
+                .unwrap(), // safe
+            bastion_details.cert.file_name().unwrap().to_str().unwrap(), // safe
+            device_details.hostname,
+            device_details.username,
+            device_details
+                .priv_key
+                .file_name()
+                .unwrap()
+                .to_str()
+                .unwrap(), // safe
+            device_details.cert.file_name().unwrap().to_str().unwrap(), // safe
+        )
+        .map_err(|err| anyhow::anyhow!("Failed to write ssh config file: {err}"))?;
+    } else {
+        writeln!(
+            &mut writer,
+            r#"Host bastion
 	User {}
 	Hostname {}
 	Port {}
```

Comment:
indention?

Reply:
We can use a \ line break (see latest commit), instead to get the full config snippet into the file as we would expect it. However, this would still keep it unindented. Alternatively, one could probably do sth. like 
```rust
concat!(
    "Hostname",
    "  User {}",
   ...
);
```